api/restapi/configure_keptn_test.go

```diff
@@ -0,0 +1,23 @@
+package restapi
+
+import (
+	"github.com/stretchr/testify/require"
+	"os"
+	"testing"
+)
+
+func Test_getEnvConfig(t *testing.T) {
+	_ = os.Setenv("MAX_AUTH_REQUESTS_PER_SECOND", "0.5")
+	_ = os.Setenv("MAX_AUTH_REQUESTS_BURST", "1")
+	config, err := getEnvConfig()
+	require.Nil(t, err)
+	require.Equal(t, 0.5, config.AuthRequestsPerSecond)
+	require.Equal(t, 1, config.AuthRequestMaxBurst)
+}
```

Comment:
this test has a race condition with the other one in this file.
Let's just unset these env vars after the test.

```suggestion
func Test_getEnvConfig(t *testing.T) {
	defer os.Unsetenv("MAX_AUTH_REQUESTS_PER_SECOND")
	defer os.Unsetenv("MAX_AUTH_REQUESTS_BURST")
	_ = os.Setenv("MAX_AUTH_REQUESTS_PER_SECOND", "0.5")
	_ = os.Setenv("MAX_AUTH_REQUESTS_BURST", "1")
	
	config, err := getEnvConfig()
	require.Nil(t, err)
	require.Equal(t, 0.5, config.AuthRequestsPerSecond)
	require.Equal(t, 1, config.AuthRequestMaxBurst)
}
```